ballerina/tests/53_server_streaming_negative_service.bal

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/log;
+
+listener Listener ep53 = new (9153);
+
+@ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_53, descMap: getDescriptorMap53()}
+service "helloWorld53" on ep53 {
```

Comment:
I guess we don't need this service here. Is there a place which we invoke this?

Reply:
This service is invoked in 53_server_streaming_negative_client